website/static/js/home-page/quickProjectSearchPlugin.js

```diff
@@ -46,7 +47,7 @@ var QuickSearchProject = {
         self.templateNodes.start();
 
         // Load up to first ten nodes
-        var url = $osf.apiV2Url('users/me/nodes/', { query : { 'embed': 'contributors'}});
+        var url = $osf.apiV2Url('users/me/nodes/', { query : { 'embed': ['contributors','root', 'parent']}});
```

Comment:
Embedding can be very expensive. @zamattiac Can you do a bit of performance testing to check that our response times aren't slowed down too much by this change?